src/board/properties/housingProperty.js

```diff
@@ -12,32 +12,26 @@ var PG_BLUE = 7;
 
 // rent should be array with following format:
 // [rent, 1 house, 2 houses, 3 houses, 4 houses, hotel]
-function HousingProperty(name, cost, propertyGroup, rent) {
+function HousingProperty(name, cost, propertyGroup, rent, houseCost) {
   Property.call(this, name, cost, propertyGroup);
   this.rent = rent;
   this.numHouses = 0;
+  this.houseCost = houseCost;
 };
 
 inherits(HousingProperty, Property);
 
 HousingProperty.prototype.performLandingAction = function(game) {
-
-    HousingProperty.super_.prototype.performLandingAction.call(this, game);
+  return HousingProperty.super_.prototype.performLandingAction.call(this, game);
 };
 
 HousingProperty.prototype.getRent = function(game) {
 
+  if (!this.owner) return 0;
```

Comment:
hm maybe this should be an assert or something b/c we would want to know about that kind of error.